src/content/en/index.html

```diff
@@ -43,101 +18,15 @@
-          <span class="icon-circle--large themed--background"><i class="icon icon-tick"></i></span>
-          <h3 class="large">Always up to date</h3>
-        </a>
-        <p>Web apps and sites are remotely hosted and up to date for 100% of your visitors by default.</p>
-      </li>
-      <li class="guides-list__item g--third theme--" style="margin-right: 0;">
-        <a href="{{site.WFBaseUrl}}/choosing-the-web/?hl={{page.langcode}}#instant-load" title="Instant load" class="themed">
-          <span class="icon-circle--large themed--background"><i class="icon icon-tick"></i></span>
-          <h3 class="large">Instant load</h3>
-        </a>
-        <p>The web can deliver apps and sites that are preloaded in less than a second.</p>
-      </li>
-      <li class="guides-list__item g--third theme--" style="margin-right: 0;">
-        <a href="{{site.WFBaseUrl}}/choosing-the-web/?hl={{page.langcode}}#independent" title="Independent" class="themed">
-          <span class="icon-circle--large themed--background"><i class="icon icon-tick"></i></span>
-          <h3 class="large">Independent</h3>
-        </a>
-        <p>The web does not censor content or disallow different payment providers and is an even playing field for all.</p>
-      </li>
-      <li class="guides-list__item g--third theme--" style="margin-right: 0;">
-        <a href="{{site.WFBaseUrl}}/choosing-the-web/?hl={{page.langcode}}#ephemeral" title="Ephemeral" class="themed">
-          <span class="icon-circle--large themed--background"><i class="icon icon-tick"></i></span>
-          <h3 class="large">Ephemeral</h3>
-        </a>
-        <p>Clicking on a link is effortless compared to installing a native app.</p>
-      </li>
-      <li class="guides-list__item g--third theme--" style="margin-right: 0;">
-        <a href="{{site.WFBaseUrl}}/choosing-the-web/?hl={{page.langcode}}#viral" title="Viral" class="themed">
-          <span class="icon-circle--large themed--background"><i class="icon icon-tick"></i></span>
-          <h3 class="large">Viral</h3>
-        </a>
-        <p>The web has the best sharing mechanism in the world, built in.</p>
-      </li>
-      <li class="guides-list__item g--third theme--" style="margin-right: 0;">
-        <a href="{{site.WFBaseUrl}}/choosing-the-web/?hl={{page.langcode}}#long-lasting" title="Long lasting" class="themed">
-          <span class="icon-circle--large themed--background"><i class="icon icon-tick"></i></span>
-          <h3 class="large">Long lasting</h3>
-        </a>
-        <p>Your web app will still run 20 years from now, even if you abandon its development.</p>
-      </li>
-    </ul>
-  </div>
-</div>
+<h1>Welcome</h1>
+<p>
+  Most users will never see this page, its replaced by DevSite's custom
+  index page and is only visible on 
+  <a href="https://web-central.appspot.com/web/">WebCentral</a> and in 
```

Comment:
This page is just a placeholder used on the Web Central staging server and is never published to Web Fundamentals because it is replaced by DevSites /web/_index.yaml file.

Because this is never published, I put this comment in, I can make the language stronger about it just being a placeholder if you want.